examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -979,12 +979,12 @@
               "mfgCode": null,
               "side": "server",
               "included": 1,
-              "storageOption": "External",
+              "storageOption": "RAM",
```

Comment:
Why?  This is going to take up space in the attr store that is completely unused...

Reply:
This is the easiest way to get the tests running for now, because the attribute exists. Otherwise I have to go through and ensure all the "not yet implemented" parts have "some kind of implementation" sufficient to keep the tests happy. I made the length 4 just in case any test tried to write more than one, but I could change the length to 1 for now.

Reply:
Please see PR #12690